src/runtime/providers/cloudinary.ts

```diff
@@ -1,111 +1,126 @@
-    opacity: 'o',
-    overlay: 'l',
-    underlay: 'u',
-    transformation: 't',
-    zoom: 'z',
-    colorSpace: 'cs',
-    customFunc: 'fn',
-    density: 'dn',
-    aspectRatio: 'ar'
-  },
-  valueMap: {
-    fit: {
-      fill: 'fill',
-      inside: 'pad',
-      outside: 'lpad',
-      cover: 'fit',
-      contain: 'scale',
-      minCover: 'mfit',
-      minInside: 'mpad',
-      thumbnail: 'thumb',
-      cropping: 'crop',
-      coverLimit: 'limit'
-    },
-    format: {
-      jpeg: 'jpg'
-    },
-    background (value: string) {
-      return convertHexToRgbFormat(value)
-    },
-    color (value: string) {
-      return convertHexToRgbFormat(value)
-    },
-    gravity: {
-      auto: 'auto',
-      subject: 'auto:subject',
-      face: 'face',
-      sink: 'sink',
-      faceCenter: 'face:center',
-      multipleFaces: 'faces',
-      multipleFacesCenter: 'faces:center',
-      north: 'north',
-      northEast: 'north_east',
-      northWest: 'north_west',
-      west: 'west',
-      southWest: 'south_west',
-      south: 'south',
-      southEast: 'south_east',
-      east: 'east',
-      center: 'center'
-    }
-  },
-  joinWith: ',',
-  formatter: (key, value) => `${key}_${value}`
-})
+// const operationsGenerator = createOperationsGenerator({
+//   keyMap: {
+//     fit: 'c',
+//     width: 'w',
+//     height: 'h',
+//     format: 'f',
+//     quality: 'q',
+//     background: 'b',
+//     rotate: 'a',
+//     roundCorner: 'r',
+//     gravity: 'g',
+//     effect: 'e',
+//     color: 'co',
+//     flags: 'fl',
+//     dpr: 'dpr',
+//     opacity: 'o',
+//     overlay: 'l',
+//     underlay: 'u',
+//     transformation: 't',
+//     zoom: 'z',
+//     colorSpace: 'cs',
+//     customFunc: 'fn',
+//     density: 'dn',
+//     aspectRatio: 'ar'
+//   },
+//   valueMap: {
+//     fit: {
+//       fill: 'fill',
+//       inside: 'pad',
+//       outside: 'lpad',
+//       cover: 'fit',
+//       contain: 'scale',
+//       minCover: 'mfit',
+//       minInside: 'mpad',
+//       thumbnail: 'thumb',
+//       cropping: 'crop',
+//       coverLimit: 'limit'
+//     },
+//     format: {
+//       jpeg: 'jpg'
+//     },
+//     background (value: string) {
+//       return convertHexToRgbFormat(value)
+//     },
+//     color (value: string) {
+//       return convertHexToRgbFormat(value)
+//     },
+//     gravity: {
+//       auto: 'auto',
+//       subject: 'auto:subject',
+//       face: 'face',
+//       sink: 'sink',
+//       faceCenter: 'face:center',
+//       multipleFaces: 'faces',
+//       multipleFacesCenter: 'faces:center',
+//       north: 'north',
+//       northEast: 'north_east',
+//       northWest: 'north_west',
+//       west: 'west',
+//       southWest: 'south_west',
+//       south: 'south',
+//       southEast: 'south_east',
+//       east: 'east',
+//       center: 'center'
+//     }
+//   },
+//   joinWith: ',',
+//   formatter: (key, value) => `${key}_${value}`
+// })
 
 const defaultModifiers = {
```

Comment:
fwiw, i default to format and quality of auto in case you want to additionally clean these up! https://github.com/colbyfayock/cloudinary-util/blob/main/packages/url-loader/src/lib/cloudinary.ts#L141

Reply:
Added :)